static/partials/settings.html

```diff
@@ -283,6 +283,31 @@
             </div>
           </div>
 
+          <div ng-show="withTurnCustomizable">
+              <legend>{{_('Turn')}}</legend>
```

Comment:
This lacks a description that it is a custom setting normally not required. Take a look how browsers allow setting of a proxy as this is pretty much the same.